code/index.html

```diff
@@ -3,62 +3,106 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Guess Who Avenger</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="favicon.ico" /> <!--https://www.favicon.cc/-->
   </head>
   <body>
+    
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Bangers&display=swap');
+    </style>
+
+      <!---- Section with game fuctions ---->
+
     <aside class="question-section">
       <button id="restart" class="outlined-button">RESTART</button>
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="images/guesswho.png"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the Avenger have:</h1>
 
+      <!---- Drop down menu ---->
+  
       <select id="questions">
-        <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
-        </optgroup>
-        <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+
+      <!---- Drop down menu default start ---->
+
+      <option selected disabled default>Select</option>
+   
+      <!---- Drop down menu content ---->
+
+                <!---- Hair ---->
+        <optgroup label="hair color">
```

Comment:
Small thing - but maybe write the headline for each attribute with capital first letter? Hair, What, Accessories and Abilites?

Reply:
Good point!